src/bin/helpers.js

```diff
@@ -30,15 +27,17 @@ function toFixedTrunc (x, n) {
 
 const whole_wits = (number, digits) => {
   const lookup = [
-    { value: 1, symbol: " pedros" },
-    { value: 1e6, symbol: " mWits" },
-    { value: 1e9, symbol: "  Wits" },
-    { value: 1e12, symbol: " KWits" },
-    { value: 1e15, symbol: " MWits" },
+    { value: 1n, symbol: "pedros" },
+    { value: 10n ** 6n, symbol: "mWits" },
+    { value: 10n ** 9n, symbol: " Wits" },
+    { value: 10n ** 12n, symbol: "KWits" },
+    { value: 10n ** 15n, symbol: "MWits" },
```

Comment:
This can be an early optimization but as it is a constant number, what do you think about writing the result of the operation instead of doing the math on every call?

```suggestion
    { value: 1n, symbol: "pedros" },
    { value: 1000000n, symbol: "mWits" }, // 10n ** 6n
    { value: 1000000000n, symbol: " Wits" }, // 10n ** 9n
    { value: 1000000000000n, symbol: "KWits" }, // 10n ** 12n
    { value: 1000000000000000n, symbol: "MWits" }, // 10n ** 15n
```

Reply:
Idk, i think it's just more readable to see the exact exponent being used on each case. Adding the exponent as a comment does not ensure the actual `value` corresponds to that exponent, forcing you "count zeros" just to make sure values have been types properly